### ISSUE TITLE
Default GraphQL arguments are not passed into setupFunctions

### ISSUE THREAD

**Issue by kouak** [apollographql/graphql-subscriptions#49]
Hello,

### Versions used
* master@341482dc982983f4904b7fee51e672820f376830

### Setup
When defining setup functions like this :
```javascript
  setupFunctions: {
    commentAdded: (options, args) => ({
      newCommentsChannel: {
        filter: comment => comment.repository_name === args.repoFullName,
      },
    }),
  },
```

I might be wrong here, since I'm not really familiar with `graphql-subscriptions` but I assume `args` represents the graphql arguments passed into the subscription query.

Considering this query :
```
subscription CommentAdded {
  commentAdded(feed: 1234) {
    ...
   }
}
```
`args` will be properly set to the expected value :
```Javascript
{ feed: 1234 }
```

However, when default arguments are defined in the schema like this :
```
type RootSubscription {
  commentAdded(
    feed: Int = 10
  ): Comment!
}
```
`args` will not be set to `{ feed: 10 }` as one would expect.

### Actual outcome
`args` is set to
```Javascript
{}
```

### Expected outcome
`args` set to
```Javascript
{ feed: 10 }
```

### Failing test
I forked and created a failing test case demonstrating the issue. See here : https://github.com/kouak/graphql-subscriptions/commit/820101af52ec86bc0ab48836a6c8f51d68935a96

Let me know if you want me to submit a PR with this failing test case.

**Comment by NeoPhi** [comment]
@Urigo Not sure how you feel about pulling in a function that isn't exported by default but handles all of the internals of arguments.

**Comment by Urigo** [comment]
@kouak fixed by @NeoPhi  in #50 , we'll release a new version think week.

**Comment by Urigo** [comment]
released as 0.3.1
